src/commands/importExportEntities/entities/contentTypes.ts

```diff
@@ -188,3 +220,23 @@ const createUpdateTypeItemReferencesFetcher = (params: UpdateTypeParams) => (typ
     .withData(patchOps)
     .toPromise();
 };
+
+const cleanElementsFromType = (client: ManagementClient, type: Type) => {
```

Comment:
cleanElementsFromWSType?

Reply:
I rewrote the method to not be WSL-specific and kept the name